components/motor/gpio/motor_encoder.go

```diff
@@ -119,6 +119,7 @@ func newEncodedMotor(
 		maxPowerPct:             motorConfig.MaxPowerPct,
 		logger:                  logger,
 		loop:                    nil,
+		motorName:               config.Name,
```

Comment:
[nit] I'd like to refactor the passed in `config` to `cfg` so we don't clash with the `config` package. But held off.

Reply:
Thanks for putting that in a separate PR! That'll be super easy to review on its own.